core/src/com/group66/game/BustaMove.java

```diff
@@ -22,6 +22,21 @@
     /** The logger. */
     public static Logger logger;
     
+    /** Create the only object of this class */
+    private static BustaMove game = new BustaMove();
+   
+    private BustaMove() {   
+    }
+    
+    /**
+     * Gets the only object available.
+     * 
+     * @return the only available object
+     */
+    public static BustaMove getGameInstance() {
+        return game;
+    }
```

Comment:
Ok, but I thought the last method was the best method, but if they all work fine, this one is also fine to me.